src/coreclr/src/jit/emit.h

```diff
@@ -1233,6 +1233,8 @@ class emitter
 #define PERFSCORE_THROUGHPUT_4C 4.0f   // slower - 4 cycles
 #define PERFSCORE_THROUGHPUT_5C 5.0f   // slower - 5 cycles
 #define PERFSCORE_THROUGHPUT_6C 6.0f   // slower - 6 cycles
+#define PERFSCORE_THROUGHPUT_7C 7.0f   // slower - 7 cycles
```

Comment:
> PerfScore numbers are according to Arm Cortex-A55 Software Optimization Guide

Is this the same baseline used for the other instruction throughputs and do we have that documented anywhere?

Reply:
@tannergooding I think so but let's ask @briansull

Reply:
It's mentioned here
https://github.com/dotnet/runtime/blob/4480bdbe66c55caceb64fc3477010435ba70c4ec/src/coreclr/src/jit/emitarm64.cpp#L12348-L12354

Reply:
Yes,  I took the latencies from the Arm Cortex-A55 Software Optimization Guide: